design-doc-template.adoc

```diff
@@ -1,122 +1,132 @@
+__<For analyses of a promotion of an existing feature to
+'preview' or 'community' stability, only list new requirements; existing
+ requirements from the feature being promoted are assumed to continue unless
+ otherwise noted in the 'Changed requirements' section. Other analyses, including
+ those for promotion to the 'default' stability level, must list all requirements.>__
 
-=== Relevant Installation Types
-// Remove the x next to the relevant field if the feature in question is not relevant
-// to that kind of WildFly installation
-* [x] Traditional standalone server (unzipped or provisioned by Galleon)
+=== Changed requirements
 
-* [x] Managed domain
+__<Only relevant for analyses of a promotion of an existing feature to
+ 'preview' or 'community stability. Other analyses should remove this section.>__
 
-* [x] OpenShift s2i
+__<For any existing requirements from the feature being promoted that are
+ being changed or removed, describe the change.>__
 
-* [x] Bootable jar
 
-== Requirements
+=== Non-Requirements
 
-=== Hard Requirements
+__<Use this section to explicitly discuss things that readers might think are required but which are not required.>__ 
 
-=== Nice-to-Have Requirements
-// Requirements in this section do not have to be met to merge the proposed functionality.
-// Note: Nice-to-have requirements that don't end up being implemented as part of
-// the work covered by this proposal should be moved to the 'Future Work' section.
+=== Future Work
 
+__<Use this section to discuss requirements that are not addressed by this proposal but which may be addressed in later proposals.>__
 
-=== Non-Requirements
-// Use this section to explicitly discuss things that readers might think are required
-// but which are not required.
-
-=== Future Work
-// Use this section to discuss requirements that are not addressed by this proposal
-// but which may be addressed in later proposals.
+__<Proposals for `preview` stability features *must* include information on expected work to be done to bring the feature to `community` stability.>__
 
 == Backwards Compatibility
 
-// Does this enhancement affect backwards compatibility with previously released
-// versions of WildFly?
-// Can the identified incompatibility be avoided?
+__<Does this enhancement affect backwards compatibility with previously released versions of WildFly? Can the identified incompatibility be avoided?>__
 
 === Default Configuration
 
+__<Does the proposed work change the default value of any current configuration attributes? Does it change the configuration generated by any current Galleon layers?>__
+
 === Importing Existing Configuration
 
+__<Does the proposed work affect the ability to run WildFly running an existing configuration? Is there anything else about the proposed work that may require changes to the WildFly server migration tool?>__
+
 === Deployments
 
+__<Does this feature change the behavior of deployments in incompatible ways?>__
```

Comment:
I think this should be more detailed, perhaps:
`__<Does this feature change the behavior of deployments in incompatible ways? If yes please detail what is the deployment issue observed when no change is done, and what is the change needed to solve the deployment issue>__`

Reply:
updated; thanks